cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -155,6 +155,120 @@ CREATE TABLE IF NOT EXISTS zipcodes (
 CREATE INDEX IF NOT EXISTS geom_index ON counties USING GIST (geom);
 CREATE UNIQUE INDEX IF NOT EXISTS zipcode_index ON zipcodes (zipcode);
 
+-- Precomputed table is required to ensure acceptable latency for the
+-- tileserver.
+CREATE TABLE IF NOT EXISTS demographics_by_state(
+    census_geo_id varchar(255) NOT NULL,
+    name varchar(255) NOT NULL,
+    black_population float,
+    white_population float,
+    total_population real,
+    under_five_population real,
+    poverty_population real,
+    geom geometry(Geometry, 3857),
+    PRIMARY KEY (census_geo_id)
+);
+CREATE INDEX IF NOT EXISTS geom_index ON demographics_by_state USING GIST (geom);
+
+-- Precomputed table is required to ensure acceptable latency for the
+-- tileserver.
+CREATE TABLE IF NOT EXISTS demographics_by_county(
+    census_geo_id varchar(255) NOT NULL,
+    name varchar(255) NOT NULL,
+    black_population float,
+    white_population float,
+    total_population real,
+    under_five_population real,
+    poverty_population real,
+    geom geometry(Geometry, 3857),
+    PRIMARY KEY (census_geo_id)
+);
+CREATE INDEX IF NOT EXISTS geom_index ON demographics_by_county USING GIST (geom);
+
+-- Precomputed table is required to ensure acceptable latency for the
+-- tileserver.
+CREATE TABLE IF NOT EXISTS demographics_by_zipcode(
+    census_geo_id varchar(255) NOT NULL,
+    zipcode varchar(255) NOT NULL,
+    black_population float,
+    white_population float,
+    total_population real,
+    under_five_population real,
+    poverty_population real,
+    geom geometry(Geometry, 3857),
+    PRIMARY KEY (census_geo_id)
+);
+CREATE INDEX IF NOT EXISTS geom_index ON demographics_by_zipcode USING GIST (geom);
+
+-- Pre-computed demographic data by state
+-- Only to be used by the function source
+INSERT INTO demographics_by_state(census_geo_id, geom, name, black_population, white_population, total_population, under_five_population, poverty_population)
+SELECT
+    states.census_geo_id as census_geo_id,
+    states.name AS name,
+    ST_Transform(states.geom, 3857) AS geom,
+    SUM(black_population) AS black_population,
+    SUM(white_population) AS white_population,
+    SUM(total_population) AS total_population,
+    SUM(under_five_population) AS under_five_population,
+    SUM(poverty_population) AS poverty_population
+FROM states LEFT JOIN demographics ON demographics.state_census_geo_id = states.census_geo_id
+GROUP BY states.census_geo_id, states.name, states.geom
+ON CONFLICT (census_geo_id) DO NOTHING;
+
+-- Pre-computed demographic data by county
+-- Only to be used by the function source
+INSERT INTO demographics_by_county(census_geo_id, geom, name, black_population, white_population, total_population, under_five_population, poverty_population)
```

Comment:
do we need insert for zip? if its coming in separate PR maybe leave a TODO above table def